packages/documentation-site/patternfly-docs/pages/featured-posts-data.js

```diff
@@ -9,13 +9,13 @@ export const featuredPostsData = {
       "https://miro.medium.com/v2/resize:fit:600/format:webp/1*FOVVqeJPmsoM6uJqsZHXZA.png",
   },
   post2: {
-    title: "UX writing mistakes as horror movie archetypes",
-    author: "Katie Edwards",
-    length: "3 min read",
-    URL: "https://medium.com/patternfly/ux-writing-mistakes-as-horror-movie-archetypes-7c30705762ae",
+    title: "From UX design to code",
+    author: "Tina Yip",
```

Comment:
```suggestion
    author: "Tina Tien",
```
IDK if she's had a recent name change/which is her preference? But in Medium at least her username is Tina Tien!